spesialist-selve/src/main/kotlin/no/nav/helse/mediator/api/OppgaveApi.kt

```diff
@@ -21,8 +23,8 @@ internal fun Route.oppgaveApi(
     val gruppeIdForRiskSaksbehandlere = UUID.fromString(riskSupersaksbehandlergruppe)
     get("/api/oppgaver") {
         val saksbehandlerOppgaver = withContext(Dispatchers.IO) {
-            val erRiskSupersaksbehandler = getGrupper().contains(gruppeIdForRiskSaksbehandlere)
-            oppgaveMediator.hentOppgaver(erRiskSupersaksbehandler)
+            val saksbehandlerType = if (getGrupper().contains(gruppeIdForRiskSaksbehandlere)) SUPER else VANLIG
```

Comment:
Gruppen heter "risikokontrolloerer", så jeg syns ikke det blir riktig å omtale det som `super` her.

Siden vi ennå ikke har flyttet annen tilgangsstyring til gruppemedlemskap så må du nok telle i frontenden for å få tall på hvor mange saker som er tilgjengelig for den som gjør kallet :((